src/components/I18n/I18n.js

```diff
@@ -0,0 +1,36 @@
+import { createI18n } from 'react-router-i18n';
+
+const locales = ['en', 'fr'];
+
+const translations = {
+  en: {
+    navbar: {
+      search: 'Search',
+      browse: 'Browse',
+      preface: 'Preface',
+    },
+    header: {
+      title: 'English-Greek Dictionary',
+      subtitle: 'A Vocabulary of the Attic Language',
+      author: 'S.C. Woodhouse',
+    },
+  },
+  fr: {
+    navbar: {
+      search: 'Cherche',
+      browse: 'Feuillete',
+      preface: 'Préface',
+    },
+    header: {
+      title: 'Dictionnaire Anglais-Grec',
```

Comment:
Since this is the title of the work, we shouldn't translate it. Same with the subtitle below.